pkg/performanceprofile/controller/performanceprofile_controller.go

```diff
@@ -370,10 +372,24 @@ func validateUpdateEvent(e *event.UpdateEvent) bool {
 // The Controller will requeue the Request to be processed again if the returned error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *PerformanceProfileReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	co, err := r.getClusterOperator()
+	if err != nil {
+		klog.Errorf("failed to get ClusterOperator: %v", err)
+		return reconcile.Result{}, err
+	}
+
+	operatorReleaseVersion := os.Getenv("RELEASE_VERSION")
+	operandReleaseVersion := operatorv1helpers.FindOperandVersion(co.Status.Versions, tunedv1.TunedOperandName)
+	if operandReleaseVersion == nil || operatorReleaseVersion != operandReleaseVersion.Version {
```

Comment:
Are there scenarios where we can end up with a long period of operand and operator version trying to match ?

Reply:
Only during upgrades when we have failing nodes.  Once an upgrade is successful, operatorReleaseVersion and operandReleaseVersion will always match.

Reply:
Also we were testing this with jmencak and MCO itself will not do changes to an MCP when there are failing nodes. We should document or link to a doc that says the proper way to handle a failing node is to remove it from the pool.